sdcm/cluster.py

```diff
@@ -3372,6 +3374,21 @@ def kill_gemini_thread(self):
                 if kill_result.exit_status != 0:
                     self.log.error('Terminate gemini on node %s:\n%s', loader, kill_result)
 
+    def create_swap_file(self):
```

Comment:
Can you move this common function to base node, we can also use that for db node & monitor?

Reply:
not critical, once we decided we need it for the db nodes, we can move it to a better place

Reply:
move to BaseNode

Reply:
moved